### PR TITLE
[merged] docs/formats: Elaborate a bit on delta from NULL usage

### PR DESCRIPTION
**Pull request by cgwalters**
See discussion on ostree-list.

### CODE REVIEW

File: docs/manual/formats.md [hunk 1]
```diff
@@ -79,7 +79,10 @@ concept of a "static delta".
 These deltas are targeted to be a delta between two specific commit
 objects, including "bsdiff" and "rsync-style" deltas within a content
 object.  Static deltas also support `from NULL`, where the client can
-more efficiently download a commit object from scratch.
+more efficiently download a commit object from scratch - this is
+mostly useful when using OSTree for containers, rather than OS images.
+For OS images, one tends to download an installer ISO or qcow2 image
+which is a single file that contains the tree data already.
```
Comment (gatispaeglis) [inline docs/manual/formats.md]: Ok, now it makses sense.

**Comment by cgwalters** [comment]
@rh-atomic-bot delegate=gatispaeglis

**Comment by gatispaeglis** [comment]
@cgwalters-bot r+ 0d112b3

**Comment by gatispaeglis** [comment]
@rh-atomic-bot r+ 0d112b3

**Pull request closed without merging**